features/features.go

```diff
@@ -93,6 +93,17 @@ const (
 	// AllowUnrecognizedFeatures is internal to the features package: if true,
 	// skip error when unrecognized feature flag names are passed.
 	AllowUnrecognizedFeatures
+<<<<<<< HEAD
+	// ExpirationMailerDontLookTwice enables a bug fix in expiration-mailer
+	// speeds up expiration-mailer processing by ensuring processed items
+	// get marked done.
+	ExpirationMailerDontLookTwice
+	// RejectDuplicateCSRExtensions enables verification that submitted CSRs do
+	// not contain duplicate extensions. This behavior will be on by default in
+	// go1.19.
+	RejectDuplicateCSRExtensions
+=======
+>>>>>>> refs/remotes/origin/main
```

Comment:
You've got a couple merge artifacts.

Reply:
d'oh that's my fault for going "oh, merge conflict in //features? generate will fix that!". Thanks!